app/src/main/java/com/darshan/notificity/AppDatabase.kt

```diff
@@ -1,3 +1,27 @@
+/*
```

Comment:
@goonerDroid Why is the MIT License being added to all the files?

Reply:
@darshanpania All new files which will be created/added will automatically have the open-source license by default..hence added the script

Reply:
@goonerDroid There is a LICENSE file in the root, we do not need to add it to each file. Please remove this. Makes it very cluttered. Let Spotless only format the code for readability for now.

Reply:
Ack